Android/app/src/main/java/com/cs4125/bikerentalapp/model/entity/BikeLocation.java

```diff
@@ -0,0 +1,5 @@
+package com.cs4125.bikerentalapp.model.entity;
+
+public interface BikeLocation {
+    String Display(String L);
```

Comment:
Again if you could make the variable name a bit more descriptive. There's no harm using the same name again. Also, variable names should always be lower case.

Method name should also start lowercase, maybe change it to display(...).